extensions/wikia/ARecoveryEngine/ARecoveryEngineApiController.class.php

```diff
@@ -8,11 +8,11 @@ public function getDelivery() {
 		$this->loadScript(new ResourceLoaderAdEngineSourcePointCSDelivery());
 	}
 
-	public function getSourcePointMessaging() {
+	public function getspmsg() {
```

Comment:
Currently, SP MMS is turned off if [wgVariable is set to false](https://github.com/Wikia/app/blob/5339152470b1d9a68f66428e27765ed49ce93c60/extensions/wikia/ARecoveryEngine/templates/ARecoveryEngineApiController_getBootstrap.mustache#L44) we are safe :) But I will write it down in ticket